ts_js_cpp_c/node-addon-api/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
This is unnecessary, as it's already declared in root folder.

Reply:
@gengjiawen thank you for the review, PR updated :+1: